boot/bootdata/hivecls.inf

```diff
@@ -26,6 +26,15 @@ HKCR,"Folder\shell\explore","BrowserFlags",0x00010001,"0x00000022"
 HKCR,"Folder\shell\explore","ExplorerFlags",0x00010001,"0x00000021"
 HKCR,"Folder\shell\explore\command","",0x00000000,"explorer.exe /e,""%1"""
 
+; Directory
+HKCR,"Directory","",0x00000000,"File Folder"
+;HKCR,"Directory\DefaultIcon","",0x00000000,"%SystemRoot%\system32\shell32.dll,-4"
+HKCR,"Directory","AlwaysShowExt",0x00000000,""
+HKCR,"Directory","NoRecentDocs",0x00000000,""
+HKCR,"Directory\shell\cmd","",0x00020000,"@%SystemRoot%\system32\shell32.dll,-162"
+HKCR,"Directory\shell\cmd","Extended",0x00000000,""
+HKCR,"Directory\shell\cmd\command","",0x00000000,"cmd /s /k cd ""%1"""
```

Comment:
Here standard Windows would use `pushd` command instead of  `cd`. Is there any reason why we could not use it there too?

Reply:
does not really matter?

Reply:
Or does it?

Reply:
`pushd` allow the previous dir (c:\windows\system32 on Windows) to be restored by `popd`, i'm not sure if it is ever used anywhere in this particular case but who know.

Reply:
![orrect](https://user-images.githubusercontent.com/2107452/68591863-0401e680-04d5-11ea-8b08-57b94ebf6181.png)

Reply:
```
ERROR_INVALID_NAME - The filename, directory name, or volume label syntax is incorrect.
```

Reply:
`pushd` works from drive 1 to drive 2, `cd` does not.

Reply:
`cmd!SetRootPath`

Reply:
Fixed.